schemas/logs/salesforce/insecure-external-assets.yml

```diff
@@ -0,0 +1,145 @@
+    description: >-
+      Type of insecure asset.
+      Possible Values: Base URI, Connect, Font,
+      Frame Ancestor: External page that embeds the Salesforce page in an iframe,
+      Frame, Image, Media, Object, Other, Plugin Types, Script, Style
+  - name: CLIENT_IP
+    required: false
+    type: string
+    indicators:
+      - ip
+    description: >-
+      The IP address of the client that’s using Salesforce services.
+      A Salesforce internal IP (such as a login from Salesforce Workbench
+      or AppExchange) is shown as "Salesforce.com IP".
+      For example: 10.0.0.1.
+  - name: CPU_TIME
+    required: true
+    type: bigint
+    description: >-
+      The CPU time in milliseconds used to complete the request.
+      This field indicates the amount of activity taking place in the app server layer.
+  - name: DOCUMENT_URI
+    required: true
+    type: string
+    description: >-
+      URL of the page that contains the insecure asset, excluding the query parameter.
+      Example: https://company.my.salesforce.com/00XXXXXXXXX
+  - name: INSECURE_URI
+    required: true
+    type: string
+    description: >-
+      Insecure external asset URL being used to load an asset insecurely.
+      For example, loading Javascript libraries using http://ajax.googleapis.com/ in your custom
+      code will log an Insecure External Asset Event with the INSECURE_URI
+      field set to this URL. Find this reference in your code and
+      update it to use https://ajax.googleapis.com/  instead.
+      Example: http://pbs.twimg.com/profile_images/5699091412070816/Z4Stwts_normal.jpeg
+  - name: LOGIN_KEY
+    required: true
+    type: string
+    description: >-
+      The string that ties together all events in a given user’s login session. It starts with a login event and
+      ends with either a logout event or the user session expiring. For example: GeJCsym5eyvtEK2I.
+  - name: ORGANIZATION_ID
+    required: true
+    type: string
+    description: "The 15-character ID of the organization. For example: 00D000000000123."
+  - name: REQUEST_ID
+    required: true
+    type: string
+    indicators:
+      - trace_id
+    description: >-
+      The unique ID of a single transaction. A transaction can contain one or more events.
+      Each event in a given transaction has the same REQUEST_ID.
+      For example: 3nWgxWbDKWWDIk0FKfF5DV.
+  - name: RUN_TIME
+    required: true
+    type: bigint
+    description: "The amount of time that the request took in milliseconds."
+  - name: SESSION_KEY
+    required: true
+    type: string
+    description: >-
+      The user’s unique session ID. You can use this value to identify all user events within a session.
+      When a user logs out and logs in again, a new session is started. For Login Event Type,
+      this field is usually null because the event is captured before a session is created.
+      Example d7DEq/ANa7nNZZVD
+  - name: TIMESTAMP
+    required: true
+    type: float
```

Comment:
should be timestamp type.

Reply:
The format of the specific timestamp is _very_ weird.
It _might_ be possible to parse it using `strftime`.
@georgepsarakis I think I have posted a possible format for this in a Slack discussion.
Can you test it against some sample using the `customlogs` dev tool?

Reply:
Perhaps you already read it but adding it for completeness: https://github.com/panther-labs/panther-analysis/pull/199#discussion_r578976369

Reply:
Yep I replied there about the output format confusion